docs/dev.md

```diff
@@ -42,5 +42,5 @@ cozy-banks
 cozy-client
 
 $ cd cozy-client/packages/cozy-client
-$ ln -s ~/code/cozy-banks/node_modules node_modules
+$ ln -s ~/code/cozy-banks/node_modules .
```

Comment:
Why ? Was it because if there was already a node_modules folder it would create the symbolic link inside `node_modules` ? Here the idea is that if it is already existing, the link will fail, so that you know that you have to remove the folder ? Maybe this can be made explicit ?

```
+ rm -rf node_modules
```